packages/babel-preset-mc-app/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = function getBabePresetConfigForMcApp() {
   const env = process.env.BABEL_ENV || process.env.NODE_ENV;
   const isEnvDevelopment = env === 'development';
   const isEnvProduction = env === 'production';
+  const jsxRuntime = process.JSX_RUNTIME || 'classic';
```

Comment:
```suggestion
  const jsxRuntime = process.env.JSX_RUNTIME || 'classic';
```

Reply:
Why on 🌎 did I remove the `env` before commiting 🤷